src/services/redisCache.ts

```diff
@@ -0,0 +1,17 @@
+import { Redis } from "ioredis";
+import { REDIS_HOST, REDIS_PASSWORD, REDIS_PORT } from "../constants";
+
+const RedisCache = new Redis({
+  host: REDIS_HOST!,
+  port: REDIS_PORT || 6379,
+  password: REDIS_PASSWORD,
+});
+
+// Setting the limit of the max memory of the cache to 100MB
```

Comment:
What do you use to determine the required size of the cache or is this a standard size?

Reply:
@Rumone Well The size of the cache for now is basically a placeholder value for now. In production we will obviously have to account for the size of the actual real data as well as the total size of the mainframe where the API is hosted since this is going to be an in-memory cache to determine the size of the Cache. 

a few months back @palisadoes had suggested that a 100GB VPS can be used for the API. So around 0.1 percent of that is 100MB and that is basically the size of the placeholder value

Do you think I should make the size a global variable so that it can be changed later?

Reply:
The value is used in one location so no need to make it a global variable.